plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/AbstractDeployPreferencesPanel.java

```diff
@@ -17,29 +17,12 @@
 package com.google.cloud.tools.eclipse.appengine.deploy.ui;
 
 import org.eclipse.core.databinding.DataBindingContext;
-import org.eclipse.swt.events.DisposeEvent;
-import org.eclipse.swt.events.DisposeListener;
 import org.eclipse.swt.widgets.Composite;
-import org.eclipse.ui.forms.FormColors;
-import org.eclipse.ui.forms.widgets.FormToolkit;
 
-abstract class DeployPreferencesPanel extends Composite {
-  private FormToolkit formToolkit;
+abstract class AbstractDeployPreferencesPanel extends Composite {
```

Comment:
There doesn't seem to be a need to rename this class. It's not necessary to use the word "Abstract" in the name of an abstract class.